### ISSUE TITLE
Would you adopt this PR?

### ISSUE THREAD

**Issue by busslina** [akvelon/dart-highlighting#74]
Would you adopt this PR?
https://github.com/git-touch/highlight.dart/pull/50

**Comment by nausharipov** [comment]
Yes. Thank you. @busslina can you please create a PR? Meanwhile, I'll request `write` access to this repo.
